scripts/qa_crowbarsetup.sh

```diff
@@ -3656,6 +3661,15 @@ function oncontroller_magnum_service_setup ()
             --disk-format qcow2 --container-format bare --public \
             --property os_distro=fedora-atomic magnum-service-image
     fi
+
+    # create magnum flavors used by tempest
+    if ! openstack flavor show magnum.master; then
+        openstack flavor create --ram 1024 --disk 8 --vcpus 1 magnum.master
+    fi
+
+    if ! openstack flavor show magnum.minion; then
+        openstack flavor create --ram 512 --disk 8 --vcpus 1 magnum.minion
```

Comment:
This is exactly the size of the m1.smaller flavor that we create in the testsetup step. Does it really make sense to have identical flavors with different names?